episodes/chapter3.md

```diff
@@ -0,0 +1,120 @@
+
+## Running a PET (privacy-enhancing technology) analysis without programming!
+
+In this episode, you will perform a PET analysis on an existing vantage6 collaborations (based on 'dummy' nodes) that resemble the two described above. For reference, the datasets of each organization can be seen here (TODO).
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 1: understanding a simple federated algorithm
+
+First, let's take a look at one of the federated algorithms -available on the vantage6's community store- that will be used in this episode: [a federated average](https://github.com/IKNL/v6-average-py/blob/master/v6-average-py/__init__.py). Based on the code and its comments:
+
+- What is the difference between the 'central_average' and the 'partial_average' functions?
+- What would happen if this algorithm is executed within a collaboration that has one of its nodes 'offline'?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 2: exploring the status of existing collaborations configured on a vantage6 server
+
+Below are the administrator credential of GHT and PhY24 collaborations (passwords will be given by the instructors).
+
+|  User  |  Roles   |  Collaboration   |
+|----|-----|-----|
+|PhY24-admin  | Collaboration Admin    |PhY24      |
+|GHT-admin  | Collaboration Admin    |GHT    |
+
+Check the status of the nodes of each collaboration. Log in to each one with the given credentials, click on 'Administration' on the top of the UI, select 'Collaborations' on the left panel, and then select the corresponding collaboration. If there are 'offline' nodes, click on the 'Nodes' panel on the left and check when these were seen for the last time.
+
+![Collaboration status](fig/chapter3/collab-status-offline.png)
+
+- Based on what you see on Challange #1, which collaboration would be ready to request the 'Average' algorithm on it?
+- For the other collaboration, which organization you would need to reach in order to fix the issue?
+
+You will now link the 'community-store' to the collaboration whose nodes are ready for it. 
+
+1. Login into the organization using the corresponding credentials from above. 
+2. Click on 'Administration' on the top of the UI, select 'Collaborations' on the left panel, and then select the corresponding collaboration.
+3. Click on '+ Add algorithm store'
+4. Add the vantage6's community store. Use any descrption as name, and provide community store URL: `https://store.cotopaxi.vantage6.ai`
+5. Make sure the store is now shown on the collaboration details:
+![Community store entry on the collaboration details](fig/chapter3/community-store-entry.png)
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 3: your first algorithm execution as a researcher
```

Comment:
Maybe we can break every step into it own challenge / answer block? So that we can hide the images?

Reply:
Done (with exception of the last one)